composer.json

```diff
@@ -8,17 +8,11 @@
     "name": "Alexandre Saelens",
     "email": "alexandre.saelens@dna.co.nz"
   }],
-  "extra": {
-    "expose": [
-      "css",
-      "javascript"
-    ]
-  },
   "minimum-stability": "dev",
   "prefer-stable": true,
   "require": {
     "dnadesign/silverstripe-elemental": ">=3",
-    "silvershop/silverstripe-hasonefield": "3.x-dev",
+    "silvershop/silverstripe-hasonefield": "3.0.4",
```

Comment:
Being a module, the version should be more flexible, eg: ^3

Reply:
fixed